code/modules/admin/verbs/adminpanelweapons.dm

```diff
@@ -97,6 +97,66 @@
 						addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(weaponhits), 2, picked_atom, point_defense), hiteta SECONDS)
 						message_admins("[key_name_admin(src)] Fired a single Railgun Slug at the Almayer at a random location, [picked_atom], with the possibility of missing as [point_defense]")
 
+		if("Orb. Def. Cannon")
+			if(exactplace == TRUE)
+				shipwide_ai_announcement("DANGER: ORBITAL DEFENSES ENGAGING, INCOMING SHOT. BRACE, BRACE, BRACE. ESTIMATED TIME: [hiteta] SECONDS." , MAIN_AI_SYSTEM, 'sound/effects/missile_warning.ogg')
+				addtimer(CALLBACK(GLOBAL_PROC, GLOBAL_PROC_REF(weaponhits), 3, mob.loc, point_defense), hiteta SECONDS)
```

Comment:
do you know that the mob's loc will always be a turf?

Reply:
Would be misuse if it wasn't, it's for an admin to aim it at somewhere in a ship (Which kinda most certainly gonna be a turf) and then firin' it